test/errors/useImportNotModule-brief.good

```diff
@@ -0,0 +1 @@
+error in useImportNotModule.chpl:8: attempted to use 'someProc' which is not a module
```

Comment:
I think quoting `'use'` might be good here to prevent confusion ("in what sense am I using someProc?")